deploy.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ "${1:-}" != "testing" && "${1:-}" != "production" ]]; then
+    echo "Please provide an environment as the first argument - either \"testing\" or \"production\"!"
+    echo "You may additionally pass in a tag suffix as second parameter that will be appended to today's date in the git and docker tags."
+    echo
+    echo "For example,"
+    echo "$ ./deploy.sh testing bugfix"
+    echo "will create tags like \"deploy-testing-2023-Aug-05-bugfix\"."
+    exit 1
+fi
+
+env="${1:-}"
+tag_suffix="${2:-}"
+if [[ -n "$tag_suffix" ]]; then
+    # add leading dash
+    tag_suffix=${tag_suffix/#/-}
+fi
+
+# Tag the currently checked-out revision with in GitHub
+date=$(date +%Y-%b-%d)
+tag="deploy-${env}-${date}${tag_suffix}"
+echo "Tagging version as $tag in git."
+git tag -a $tag -m "Deployment to ${env}" && git push origin $tag
+
+# Build the image for the Django app
+docker compose --env-file .env.${env} build
+
+# Export the images
+docker save cpmonitor:${env} -o cpmonitor.tar
+docker save klimaschutzmonitor-dbeaver -o klimaschutzmonitor-dbeaver.tar
```

Comment:
Why does dbeaver have no env tag here but has one in "docker tag klimaschutzmonitor-dbeaver:${env}"?

Reply:
Fixed it.